From 18471cf98e150070abd695269d68347ebba10dee Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Fri, 28 Sep 2018 11:25:19 -0500
Subject: [PATCH] remove /ZI flag on win (incompatible with /GL)

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9755ab4..f5023a9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -205,12 +205,12 @@ else()
     # /Wall = all warnings
 
     add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:/Ox>")
-    add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:/ZI>")
+    # add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:/ZI>")
 
     add_compile_options("$<$<CONFIG:RELEASE>:/Ox>")
     add_compile_options("$<$<CONFIG:RELEASE>:/D>")
     add_compile_options("$<$<CONFIG:RELEASE>:/NDEBUG>")
-    add_compile_options("$<$<CONFIG:RELEASE>:/ZI>")
+    # add_compile_options("$<$<CONFIG:RELEASE>:/ZI>")
 
     add_compile_options("$<$<CONFIG:DEBUG>:/Od>")
 
-- 
2.17.1

